addons/skin.estuary/1080i/DialogAudioDSPManager.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<window type="dialog">
+<window>
 	<zorder>1</zorder>
```

Comment:
This zorder might also be some leftover? I wouldnt know what it should be good for..